Rakefile

```diff
@@ -109,6 +131,25 @@ task :release do
   puts 'You can find directions here: https://github.com/blog/1547-release-your-software'
   puts
   puts 'Make sure you add release notes!'
+  cp ::File.join(base_dir, "#{name}-#{version}.tgz"), pkg_dir
+
+  latest_release = github_client.latest_release(github_repo)
```

Comment:
What does this return if the repo doesn't have any releases? Do we need to do something like
```ruby
latest_release ||= { name: 'master' }
```